### DISCUSSION TITLE
Errors when using the Theia plugin generator (hello world front end example)

### DISCUSSION THREAD

**Discussion by sboudreaux96** [eclipse-theia/theia#9489] [Q&A]
Hello - I am trying to run the theia plugin generator using yo @theia/plugin. After selecting all of the options (including the hello world template with the front end configuration and it throws the below error. I have used the same generator to generate a back end plugin and that works fine but the front end configuration doesnt work.

ERROR in C:\Users\username\theia_test\theia\node_modules\dom-serializer\lib\index.d.ts
[tsl] ERROR in C:\Users\username\theia_test\theia\node_modules\dom-serializer\lib\index.d.ts(1,13)
      TS1005: '=' expected.

ERROR in C:\Users\username\theia_test\theia\node_modules\dom-serializer\lib\index.d.ts
[tsl] ERROR in C:\Users\username\theia_test\theia\node_modules\dom-serializer\lib\index.d.ts(1,27)
      TS1005: ';' expected.

ERROR in C:\Users\username\theia_test\theia\node_modules\domutils\lib\manipulation.d.ts
[tsl] ERROR in C:\Users\username\theia_test\theia\node_modules\domutils\lib\manipulation.d.ts(1,13)
      TS1005: '=' expected.

ERROR in C:\Users\username\theia_test\theia\node_modules\domutils\lib\manipulation.d.ts
[tsl] ERROR in C:\Users\username\theia_test\theia\node_modules\domutils\lib\manipulation.d.ts(1,36)
      TS1005: ';' expected.

**Comment by vince-fugnitto** [comment]
@sboudreaux96 is it related to https://github.com/eclipse/theia-generator-plugin/issues/33?

**Reply by sboudreaux96** [reply]
@vince-fugnitto  It is some-what related to that but I am using a frontend plugin while that issue uses a backend plugin. As stated in the question above the backend plugin works perfectly but this issue comes up only with the frontend.

**Reply by vince-fugnitto** [reply]
> @vince-fugnitto It is some-what related to that but I am using a frontend plugin while that issue uses a backend plugin. As stated in the question above the backend plugin works perfectly but this issue comes up only with the frontend.

You can open an issue directly on that repository (I do not maintain it myself) if you are experiencing issues (https://github.com/eclipse/theia-generator-plugin/issues). Alternatively you can also write a vscode extension instead.
